W/WCPG/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WCPG"
+version = v"0.9.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/fixif/WCPG/archive/refs/tags/0.9.tar.gz", "4f6b1d2abc298891ae9e3966428c2d8b4e8bbc3528e917d261d8613dea41ab7d"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+ln -s ${libdir}/libopenblas.${dlext} ${libdir}/libblas.${dlext}
+ln -s ${libdir}/libopenblas.${dlext} ${libdir}/liblapack.${dlext}
+cd $WORKSPACE/destdir/include/
+wget https://www.netlib.org/clapack/f2c.h
+cd $WORKSPACE/srcdir/WCPG-0.9/
+chmod +x autogen.sh 
+./autogen.sh 
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
```

Comment:
Doesn't it work on other platforms?

Reply:
Thank you for the feedback, I'll take a look at it and correct everything tomorrow. It does not work on other platforms and I have no clue why, yet. I'll take a look at it tomorrow too.
Tbh I'm already glad it partially worked... It took me quite some time to figure out that `OpenBLAS32` instead of `OpenBLAS` solved most of the issues.